packages/teleport/src/Kubes/ConnectDialog/ConnectDialog.story.test.tsx

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
i would also add a test for the other case: either a storybook snapshot test or a render and look for the specific output test

Reply:
Added a story for sso case and applied a snapshot test.